docs/user-guide/dream/dream-data-reduction.ipynb

```diff
@@ -91,7 +93,8 @@
     "    *powder.providers,\n",
     ")\n",
     "\n",
-    "pipeline = sciline.Pipeline(providers, params=params)"
+    "pipeline = sciline.Pipeline(providers, params=params)\n",
+    "pipeline = powder.set_pixel_mask_filenames(pipeline, [])"
```

Comment:
It is not great that this always needs to be called even when there are no file names. Can this be done with a default parameter?

Reply:
I agree, but it cannot, unfortunately (since the `Optional` this is replacing is a couple of nodes downstream of the filename).

Alternatives:
- Add parameter to the workflow creation, such `DreamWorkflow(pixel_masks=[])`.
- Set a default for `DetectorMasks`, which has the pitfall that later setting `PixelMaskFilename` siltently has no effect (since setting the default for `DetectorMasks` trimmed the branch of providers leading to `PixelMaskFilename`. `set_pixel_mask_filenames` could re-add this branch, but the pitfall remains.

Reply:
What about setting `PixelMaskFilename = []` by default?

Reply:
You mean as a default param in `DreamWorkflow`? That can be done, but mapping again differently afterwards is not possible.